recipes/hdf4/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [[ $(uname) == Darwin ]]; then
+  export DYLD_FALLBACK_LIBRARY_PATH=$PREFIX/lib
+fi
+
+export CXXFLAGS="${CFLAGS}"
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
+export CFLAGS="${CFLAGS} -pipe -O2 -fPIC -I${PREFIX}/include"
+
+chmod +x configure
+
+# The --enable-silent-rules is needed because Travis CI dies on the long output from this build.
+./configure \
+    --disable-static \
```

Comment:
Should we be disabling this? Static libraries are nice sometimes. Mainly thinking of end user that consume these for building an application for instance.